src/WaterPokemon.java

```diff
@@ -0,0 +1,40 @@
+public class WaterPokemon extends PokemonSuper {
+
+    private int water;
+    private int superWater;
+
+    public WaterPokemon(String name, int level, int hp, int xp, int water, int superWater) {
+        super(name, level, hp, xp);
+        this.water = water;
+        this.superWater = superWater;
+    }
+
+    public void hydroCanon(){
+        System.out.println("You wer shot with a Hydro Cannon, lose 100 health");
+    }
+
+    public void hydroPump(){
+        System.out.println("Hydropump knocks you out");
+    }
+
+    @Override
+    public void setName(String name) {
```

Comment:
Hier overschrijf je een methode, maar het voegt niks toe. Het enige wat je hier doet is de super aanroepen. Als je deze methode niet zou overschrijven, zou de werking hetzelfde blijven.